internal/pkg/cli/init.go

```diff
@@ -485,6 +485,7 @@ func (o *initOpts) deployEnv() error {
 		// Set the application name from app init to the env init command, and check whether a flag has been passed for envName.
 		initEnvCmd.appName = *o.appName
 		initEnvCmd.name = o.initVars.envName
+		initEnvCmd.federatedSession = true
```

Comment:
I think we should probably keep it consistent with the default value in `env init` which is `false`. I would assume relatively few users would need to set this and they could use `env init` instead of `init`.